src/rust/sysmon-subgraph-generator/src/models/process/create.rs

```diff
@@ -0,0 +1,83 @@
+use crate::models::{get_image_name, strip_file_zone_identifier, utc_to_epoch};
+use graph_descriptions::file::FileState;
+use graph_descriptions::graph_description::*;
+use graph_descriptions::node::NodeT;
+use graph_descriptions::process::ProcessState;
+use sysmon::ProcessCreateEvent;
+
+/// Creates a subgraph describing a `ProcessCreateEvent`.
+///
+/// Subgraph generation for a `ProcessCreateEvent` includes the following:
+/// * An `Asset` node - indicating the asset in which the process was created
+/// * A parent `Process` node - indicating the process that created the subject process
+/// * A subject `Process` node - indicating the process created per the `ProcessCreateEvent`
+/// * A process `File` node - indicating the file executed in creating the new process
+pub fn generate_process_create_subgraph(
+    process_start: &ProcessCreateEvent,
+) -> Result<Graph, failure::Error> {
+    let timestamp = utc_to_epoch(&process_start.event_data.utc_time)?;
+    let mut graph = Graph::new(timestamp);
+
+    let asset = AssetBuilder::default()
+        .asset_id(process_start.system.computer.computer.clone())
+        .hostname(process_start.system.computer.computer.clone())
+        .build()
+        .map_err(|err| failure::err_msg(err))?;
+
+    let parent = ProcessBuilder::default()
+        .asset_id(process_start.system.computer.computer.clone())
+        .state(ProcessState::Existing)
+        .process_id(process_start.event_data.parent_process_id)
+        .process_name(get_image_name(&process_start.event_data.parent_image.clone()).unwrap())
+        .process_command_line(&process_start.event_data.parent_command_line.command_line)
+        .last_seen_timestamp(timestamp)
+        //        .created_timestamp(process_start.event_data.parent_process_guid.get_creation_timestamp())
```

Comment:
porquoi non

Reply:
This was from the original code: 

https://github.com/grapl-security/grapl/blob/staging/src/rust/sysmon-subgraph-generator/src/main.rs#L125

I figured we wanted to keep it for some reason